packages/core/src/components/Links/_Links.docs.scss

```diff
@@ -0,0 +1,51 @@
+/*
+Links
+
+Links connect users to a different page or further information.
+
+@uswds https://standards.usa.gov/components/labels/
+
+Markup: Links.example.html
+
+Style guide: components.links
+*/
+
+/*
+---
+
+### When to use
+
+- Navigating to a new page or view in your application
```

Comment:
One way to tighten the technical boundaries is: If the URL changes, use a link, otherwise use a button.

There may be exceptions to that, but I can't think of any right now.

Reply:
Updated